tests/e2e/calculator-engine.spec.js

```
@@ -142,6 +142,87 @@ test('keeps duration keyboard steps aligned with its observable value', async ({
   await expect(page.locator('#duration-value')).toHaveText('0.25 years');
 });
 
+test('keeps the live estimate clickable and every research range curve usable', async ({
+  page,
+}) => {
+  await page.goto('/');
+  await page.waitForFunction(() => Boolean(window.calculator));
```

ESLint

[error] 'window' is not defined.

(no-undef)